targets/REV3-LVSS/main.cpp

```diff
@@ -91,11 +91,13 @@ int main() {
         log::LOGGER.log(log::Logger::LogLevel::INFO, "Connected to CAN network\r\n");
     }
 
-    // Initialize all the CANOpen drivers.
+    // Initialize all the CANOpen dev.
     IO::initializeCANopenDriver(&canOpenQueue, &can, &timer, &canStackDriver, &nvmDriver, &timerDriver, &canDriver);
 
+    // LVSS::LVSS lvss = LVSS::LVSS(lvssEn0, lvssEn1, lvssEn2);
+
     // Initialize the CANOpen node we are using.
-    IO::initializeCANopenNode(&canNode, &lvss, &canStackDriver, sdoBuffer, appTmrMem);
+    // IO::initializeCANopenNode(&canNode, &lvss, &canStackDriver, sdoBuffer, appTmrMem);
```

Comment:
Why is this commented out?

Reply:
Was waiting to finalize LVSS struct and its constructor, uncommented it now.